test/conftest.py

```diff
@@ -265,6 +265,12 @@ def pudl_settings_fixture(request, tmpdir_factory,  # noqa: C901
     pudl_in = request.config.getoption("--pudl_in")
     pudl_out = request.config.getoption("--pudl_out")
 
+    # Having this in a constant place will allow us to cache inputs which should
+    # speed up the tests considerably
+    if os.environ["GITHUB_ACTIONS"]:
+        pudl_in = pathlib.Path(os.environ["HOME"]) / "pudl-work"
+        pudl_out = pathlib.Path(os.environ["HOME"]) / "pudl-work"
```

Comment:
this seems very reasonable, but it also feels a bit o/s of our normal context of how we set pudl_in and pudl_out. just a comment, you don't need to justify or respond.

Reply:
Yeah, I needed it to be in a fixed location for caching purposes.